src/app/server/Mdns.cpp

```diff
@@ -144,6 +151,76 @@ CHIP_ERROR AdvertiseCommisioning()
     }
     advertiseParameters.SetShortDiscriminator(static_cast<uint8_t>(value & 0xFF)).SetLongDiscrimininator(value);
 
+    if (DeviceLayer::ConfigurationMgr().GetDeviceType(value) != CHIP_NO_ERROR)
+    {
+        ChipLogProgress(Discovery, "DNS-SD Device Type not set");
```

Comment:
nit: are we sure it is not set? since error could be anything maybe a more correct message would be 'Device type not available'.

same below.